overlord/state/change.go

```diff
@@ -520,6 +520,33 @@ func (c *Change) AbortLanes(lanes []int) {
 	c.abortLanes(lanes, make(map[int]bool), make(map[string]bool))
 }
 
+// AbortUnreadyLanes aborts the tasks from lanes that aren't fully ready, where
+// a ready lane is one in which all tasks a ready.
```

Comment:
nitpick: s/a ready/are ready/

Reply:
I'll fix that in #11520

Reply:
Actually fixed it here since I was already pushing a patch for a comment Pawel made.